httpdocs/ajax/server/list.php

```diff
@@ -12,7 +12,9 @@ function error($message) {
   $mysqli->set_charset('utf8');
   $offset = isset($data->offset) ? intval($data->offset) : 0;
   $limit = isset($data->limit) ? intval($data->limit) : 10;
-  $query = "SELECT * FROM server ORDER BY `share` - `load` DESC LIMIT $limit OFFSET $offset";
+  $branch = basename(dirname(dirname(dirname(dirname(__FILE__)))));
+  $extra_condition = "JOIN server_branch ON server.id=server_branch.id WHERE branch=\"$branch\"";
+  $query = "SELECT * FROM server $extra_condition ORDER BY `share` - `load` DESC LIMIT $limit OFFSET $offset";
```

Comment:
It's not a condition, but a join:
```suggestion
  $join= "JOIN server_branch ON server.id=server_branch.id WHERE branch=\"$branch\"";
  $query = "SELECT * FROM server $join ORDER BY `share` - `load` DESC LIMIT $limit OFFSET $offset";
```